dashbuilder/dashbuilder-backend/dashbuilder-services/src/main/java/org/dashbuilder/renderer/service/RendererSettingsServiceImpl.java

```diff
@@ -1,19 +1,36 @@
 package org.dashbuilder.renderer.service;
 
 
+import javax.annotation.PostConstruct;
 import javax.enterprise.context.ApplicationScoped;
 
+import org.dashbuilder.renderer.RendererSettings;
 import org.jboss.errai.bus.server.annotations.Service;
 
 @Service
 @ApplicationScoped
 public class RendererSettingsServiceImpl implements RendererSettingsService {
 
-    String DEFAULT_RENDERER_PROPERTY = "org.dashbuilder.renderer.default";
+    private RendererSettings settings;
+    
+    @PostConstruct
+    public void loadSettings() {
```

Comment:
Hello @jhrcek thanks again for your valuable feedback. I will address all the changes in a next commit.

Regarding the error, I am not able to reproduce it! I tried with and without internet connection but  all system properties combination worked. Are you sure you started the server with:

`./standalone.sh -c standalone-full.xml -Dorg.dashbuilder.renderer.default=c3`

If you started it with gwtcharts and in an offline environment than the issue is expected because gwtcharts will take place instead c3 and it can't work offline. However, if you do offline = true and try to use gwtcharts, it should not be a problem, because gwtcharts won't be default because it will be removed.

Thanks!

Reply:
@jhrcek  I added a new commit with the suggested changes, thanks! If you find something wrong just let me know and I will update the PR.